core/state_processor.go

```diff
@@ -118,26 +98,31 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 // and uses the input parameters for its environment. It returns the receipt
 // for the transaction, gas used and an error if the transaction failed,
 // indicating the block was invalid.
-func ApplyTransaction(config *params.ChainConfig, bc ChainContext, author *common.Address, gp *GasPool, statedb *state.StateDB, header *types.Header, tx *types.Transaction, usedGas *uint64, cfg vm.Config, gcWl *GasCurrencyWhitelist, regAdd *RegisteredAddresses, gasPriceMinimum *big.Int, infraFraction *InfrastructureFraction) (*types.Receipt, uint64, error) {
+func ApplyTransaction(config *params.ChainConfig, bc ChainContext, author *common.Address, gp *GasPool, header *types.Header, statedb *state.StateDB, tx *types.Transaction, usedGas *uint64, cfg vm.Config, gasPriceMinimum *big.Int, infraFraction *gpm.InfrastructureFraction) (*types.Receipt, uint64, error) {
```

Comment:
I see that you've changed the ordering of statedb and header here (and in other places as well). I understand that your intent is consistency.  

However,  note that you are changing the ordering of the original geth code.  So whenever we merge upstream geth into our code, we may need to resolve conflicts.

Reply:
Err great point - didn't think about this. Do you have a preference? 

On the one hand, I prefer upstream compatibility over consistency; however, this conflict would be minor.

Reply:
Ya, I prefer upstream compatibility as well.  Although I don't have a very strong one.

Reply:
I'm going to stick with this because we get consistency but, because we've been changing args anyways, we don't avoid conflicts in function signatures by reverting it. Work for you?

Reply:
Ya, that works for me.